scripts/utility-wfrp4e.js

```diff
@@ -636,9 +636,10 @@ class WFRP_Utility
     {
       let percentage = 0;
       let matchCounter = 0;
-      for (let i = 0; i < key.length; i++)
+      let myword = object[key].toLowerCase();
```

Comment:
If this is lowercase, shouldn't query be as well?

Reply:
Yes, but in fact in the code the upper functions are doing it.
This why i "lowercased" it. But for more security, i am adding it to the query also in the function itself.